components/apimgt/org.wso2.carbon.apimgt.internal.service/src/gen/java/org/wso2/carbon/apimgt/internal/service/GatewaySynapseArtifactsApi.java

```diff
@@ -0,0 +1,50 @@
+package org.wso2.carbon.apimgt.internal.service;
+
+import org.wso2.carbon.apimgt.internal.service.dto.ErrorDTO;
+import org.wso2.carbon.apimgt.internal.service.dto.SynapseArtifactListDTO;
+import org.wso2.carbon.apimgt.internal.service.GatewaySynapseArtifactsApiService;
+import org.wso2.carbon.apimgt.internal.service.impl.GatewaySynapseArtifactsApiServiceImpl;
+import org.wso2.carbon.apimgt.api.APIManagementException;
+
+import javax.ws.rs.*;
```

Comment:
Import the exact Class rather than using wildcard.

Reply:
This is an auto-generated file. We don't need to modify this.